modules/monitoring-granting-a-user-access-to-metrics.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * monitoring/user-workload-monitoring.adoc
+
+[id="granting-a-user-access-to-metrics_{context}"]
+= Granting a user access to metrics
+
+By default, only cluster administrator users have access to user workload metrics. Using the soft tenancy feature, you can also grant metrics access to an arbitrary user, who only has access to a particular project.
+
+.Prerequisites
+
+* You need to have deployed the workload that you want to monitor, such as an application or a service. This example uses the application described in xref:../monitoring/user-workload-monitoring.adoc#deploying-a-sample-application[Deploying a sample application].
+* You need to have user workload monitoring enabled.
+* You need to have a user created. For the sake of an example, you can create the `johnsmith` user by running `oc create user johnsmith`.
+
+.Procedure
+
+. In the Web console, navigate to *User Management* -> *Role Bindings* -> *Create Binding*.
```

Comment:
see from https://pasteboard.co/IMygdp7.png
there are two "Binding Type" types, need to specify which type we choose, and value for "Role Binding name" is required, also we should specify which Subject we choose, there are 3 values: User, Group, Service Account